qdrant-landing/content/articles/sparse-vectors.md

```diff
@@ -0,0 +1,161 @@
+
+BM25's capabibilities are well proven, but it has its limitations. For example, it often needs building complex dictionary of synonyms, which is where sparse vectors come in. Sparse vectors excel in handling large text data, making them crucial in modern data processing and an improvement over more well known methods like BM25.
+
+# Understanding Sparse Vectors
+Sparse vectors are like the Marie Kondo of data—keeping only what sparks joy (or relevance, in this case). 
+
+Sparse Vectors are a representation where each dimension corresponds to a word or subword, greatly aiding in interpreting document rankings. This clarity is why sparse vectors are essential in modern search and recommendation systems, offering an advantage over embedding or dense vectors. 
+
+Dense vectors from models like OpenAI Ada-002 or Sentence Transformers contain non-zero values for every element, whereas sparse vectors focus on relative word frequencies per document, with most values being zero. This results in a more efficient and interpretable system, especially in text-heavy applications like search.
+
+# SPLADE
+
+Let's checkout [SPLADE](https://europe.naverlabs.com/research/computer-science/splade-a-sparse-bi-encoder-bert-based-model-achieves-effective-and-efficient-full-text-document-ranking/), an excellent way to make sparse vectors. Let's look at some numbers first. Higher is better:
+
+| Model              | MRR@10 (MS MARCO Dev) | Type           |
+|--------------------|---------|----------------|
+| BM25               | 0.184   | Sparse |
+| TCT-ColBERT        | 0.359   | Dense  | 
+| doc2query-T5 [link](https://github.com/castorini/docTTTTTquery)  | 0.277   | Sparse |
+| SPLADE             | 0.322   | Sparse |
+| SPLADE-max         | 0.340   | Sparse |
+| SPLADE-doc         | 0.322   | Sparse |
+| DistilSPLADE-max   | 0.368   | Sparse |
+
+All numbers are from [SPLADEv2](https://arxiv.org/abs/2109.10086). MRR is [Mean Reciprocal Rank](https://www.wikiwand.com/en/Mean_reciprocal_rank#References), a standard metric for ranking. [MS MARCO](https://microsoft.github.io/MSMARCO-Passage-Ranking/) is a dataset for evaluating ranking and retrieval for passages.  
+
+SPLADE is quite flexible as a method, with regularization knobs that can be tuned to obtain [different models](https://github.com/naver/splade) as well: 
+
+> SPLADE is more a class of models rather than a model per se: depending on the regularization magnitude, we can obtain different models (from very sparse to models doing intense query/doc expansion) with different properties and performance. 
+
+First, let's look at how to create a sparse vector. Then, we'll look at the concepts behind SPLADE.
+
+# Creating a Sparse Vector
+
+We'll explore two different ways to create a sparse vector. The first is a simple way to create a sparse vector from the same model for both document and query. The second is higher performance way to create a sparse vector from dedicated document and query encoders:
```

Comment:
creating sparse vectors from dedicated document? will the splade customize according to the doc.

Reply:
dedicated encoders for document and query